cryptography/src/bls12381/primitives/variant.rs

```diff
@@ -129,16 +184,9 @@ impl Variant for MinSig {
     const PROOF_OF_POSSESSION: DST = G1_PROOF_OF_POSSESSION;
     const MESSAGE: DST = G1_MESSAGE;
 
-    fn sign(private: &Scalar, dst: DST, message: &[u8]) -> Self::Signature {
```

Comment:
Both `sign` and `verify` didn't need to be in `Variant` (didn't have variant-specific code).